server_unix_test.go

```diff
@@ -0,0 +1,87 @@
+//go:build !windows && !plan9
+// +build !windows,!plan9
```

Comment:
🤔 These tests should be able to run under plan9 as well, right? Might be better than trying to add a-whole-nother `server_plan9_test.go`. Maybe this file should be `server_nowindows_test.go`?